hash/mz700_cass.xml

```diff
@@ -2842,6 +2842,17 @@ BASIC 5Z-008 V1.0B
 			</dataarea>
 		</part>
 	</software>
+	
+	<software name="bhopper">
+		<description>Building Hopper</description>
+		<year>1988</year>
+		<publisher>Taku Ito.</publisher>
```

Comment:
Why the trailing `.` after the person's name?

Reply:
It is stated like that in the game itself.